src/service/utils/date.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Converts a date object into a string of the format yyyymmddHHMMss
+ * @param date A Date object
+ * @returns {string} String of the format yyyymmddHHMMss
+ */
+function formatDate(date: Date): string {
```

Comment:
probably `src/service/utils` folder is not appropriated one, what about `src/utils`? :thinking: 
Additionally... since it is `formatDate` method, what if we change method contract and wait for optional format date argument, like
`function formatDate(date: Date, dateFormat = "YYYYMMDDHHmmss"): string {`
we can use moment for instance (but up to you) `(moment(date)).format(dateFormat)`